src/element.ts

```diff
@@ -131,6 +132,17 @@ export abstract class Element {
   /** Draws an element. */
   abstract draw(element?: Element, x_shift?: number): void;
 
+  /** Gets SVGElement.  */
+  getSvgElement(): SVGElement | undefined {
```

Comment:
Unclear why we have this getter/setter here. We have `setAttribute` and `getAttribute` that maintains the indexes too.

Reply:
I wanted to avoid the usage of `any` in `setAttribute` `getAttribute`. I believe also that we have a mixture of things in `attrs` that not fit together. Normally `attrs` are only strings.

Reply:
Gotcha, it's okay for `attrs` to be `string -> any` -- it's intentional and allows users to attach arbitrary objects to elements (which they already do.)

Reply:
However the `attrs.classes` have their our methods: `addClass`, `removeClass`, `hasClass`.
Actions:
1 Remove `get/setSvgElement`
2 Change `get/setAttribute` to `any`
@0xfe do you want both actions or only 2?

Reply:
done